input/OFCB/test1/pb1.json

```diff
@@ -0,0 +1,93 @@
+{
+	"materials": {
+		"pwr_uox": {"attributes": {"recipe": true}, 
+			"constraints": [ 
```

Comment:
Cool thanks.  Can you now reduce to a normal number of space.  For JSON it is traditional to use 1, 2, or 4 spaces for indentation.  Thanks!